core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9327,4 +9327,20 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .withBigQuery()
         .ok(expectedBQSql);
   }
+
+  @Test public void testTimeAdd() {
```

Comment:
add test case for interval function you have added.

Reply:
this test case consist of both the function TImeADD as well as Interval Seconds because this function time add is  a combination .so thats why i have not written seperate testcase

Reply:
as discussed please remove the function if not required.

Reply:
resolved

Reply:
resolved